content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -0,0 +1,371 @@
+======
+Brazil
+======
+
+.. |IAP| replace:: :abbr:`IAP (In-App-Purchase)`
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+
+Introduction
+============
+
+With the Brazilian localization you can automatically compute sales taxes for goods using Avatax
+(Avalara) through API calls and you can also configure taxes for services.
+
+For the goods tax computation part, you need to configure the contacts, company, products, and
+create an account in AvaTax from the Odoo general settings.
+
+For the services taxes, you can create and configure them from Odoo directly without the need of
+computing them with AvaTax.
+
+The localization also includes a chart of account and taxes created as a template that can be
+modified if needed.
+
+Configuration
+=============
+
+Modules installation
+--------------------
+
+:ref:`Install <general/install>` the following modules to get all the features of the Brazilian
+localization:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Brazilian - Accounting`
+     - `l10n_br`
+     - Default :ref:`fiscal localization package <fiscal_localizations/packages>` adds accounting
+       characteristics for the Brazilian localization, which represent the minimum configuration
+       required for a company to operate in Brazil. The module's installation automatically loads:
+       chart of accounts, taxes, and include required fields to properly configure the contact.
+   * - :guilabel:`Brazil - Accounting Reports`
+     - `l10n_br_reports`
+     - Includes all the technical and functional requirements to generate and validate
+       :doc:`electronic documents
+       </applications/finance/accounting/customer_invoices/electronic_invoicing>`, based on the
+       technical documentation published by the SRI. The authorized documents are: invoices, credit
+       notes, debit notes, withholdings and purchase liquidations.
```

Comment:
```suggestion
       </applications/finance/accounting/customer_invoices/electronic_invoicing>`, adds a simple Tax Report that helps check the tax amount per tax group in a given period of time. Also adds the P&L and BS adapted for the Brazilian market.
```

Can you help with this correction @samueljlieber as I made a mistake and kept with the wrong info here.
Thanks!